python/mach_bootstrap.py

```diff
@@ -131,45 +111,32 @@ def install_virtual_env_requirements(project_path: str, python: str, virtualenv_
     requirements_hash = requirements_hasher.hexdigest()
 
     if marker_hash != requirements_hash:
-        print(" * Upgrading pip...")
-        _process_exec([python, "-m", "pip", "install", "--upgrade", "pip"])
-
         print(" * Installing Python requirements...")
-        _process_exec([python, "-m", "pip", "install", "-I",
-                       "-r", requirements_paths[0],
-                       "-r", requirements_paths[1],
-                       "-r", requirements_paths[2]])
+        pip_install_command = ["uv", "pip", "install"]
+        for requirements in requirements_paths:
+            pip_install_command.extend(["-r", requirements])
```

Comment:
Interesting - does passing multiple files to `-r` also work for the official pip install?

Reply:
`requirements` here represents a single file in `requirement_paths` so we append one `-r <file>` for each file.

Reply:
Ah, somehow I missed the `for` loop